aptos-move/framework/src/natives/code.rs

```diff
@@ -63,6 +65,11 @@ pub struct PackageRegistry {
     pub packages: Vec<PackageMetadata>,
 }
 
+impl OnChainConfig for PackageRegistry {
+    const MODULE_IDENTIFIER: &'static str = "code";
```

Comment:
where do we use this? i.e. don't see access anywhere

Reply:
`OnChainConfig::access_path()`